e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/record-correction-history-no-corrections.spec.js

```diff
@@ -40,6 +40,8 @@ context('Record correction history page - no corrections', () => {
 
   describe('when navigating to the page', () => {
     it('should display the tab heading and text', () => {
+      cy.assertText(utilisationReportPage.recordCorrectionHistoryTabLink(), 'Record correction history');
```

Comment:
Nit pick - can we move this assertion after the heading, to match the description